contracts/VotingToManageEmissionFunds.sol

```diff
@@ -159,20 +163,20 @@ contract VotingToManageEmissionFunds is VotingTo {
         uint256 _emissionReleaseTime, // unix timestamp
         uint256 _emissionReleaseThreshold, // seconds
         uint256 _distributionThreshold, // seconds
-        address _emissionFunds
+        address _emissionFunds,
+        uint256 _minBallotDuration
```

Comment:
I forgot one more thing: `_minBallotDuration` here must be less than the `_distributionThreshold`. Otherwise, a ballot cannot be created.

Let's also note in a comment that `_minBallotDuration` is defined in seconds and make this parameter fourth (not fifth, for security reasons, because the length of the `address _emissionFunds` parameter can be a float, with leading zeros).